build.gradle.kts

```diff
@@ -39,16 +40,6 @@ val shouldRunAllCheckers by extra(isCI || project.hasProperty("runAllCheckers"))
 
 tasks.register<UpdateEapBuildNumber>("updateEapBuildNumber")
 
-tasks.withType<DependencyUpdatesTask> {
```

Comment:
No longer needed, I've switched from `com.github.ben-manes.versions` Gradle plugin to `se.ascp.gradle.gradle-versions-filter`, which is almost identical but sets up this piece as a ready default